policy/modules/system/init.te

```diff
@@ -78,10 +82,8 @@ mls_trusted_object(initctl_t)
 
 type initrc_t, init_script_domain_type, init_run_all_scripts_domain;
 type initrc_exec_t, init_script_file_type;
-domain_type(initrc_t)
-domain_entry_file(initrc_t, initrc_exec_t)
+init_daemon_domain(initrc_t, initrc_exec_t)
```

Comment:
`init_daemon_domain()` is for long-running processes. `initrc_t` is not long-running. It is also for transitions _out of_ `initrc_t`. `init_domain()` may be acceptable.